### PR TITLE
timeseries_anomaly_detection : "a little less python looping, a little more pandas action."

### PR DESCRIPTION
**Pull request by RobAltena**
I really enjoyed studying this new sample. This PR does not change any of the Keras functionality of the sample.
In the data pipeline I simplified the code by using pandas time series and charting features. Hopefully this makes the code easier to understand and follow.

Please let me know if the changes are good, and I will add the generated files.

Signed-off-by: Robert Altena <Rob@Ra-ai.com>

### CODE REVIEW

File: examples/timeseries/timeseries_anomaly_detection.py [hunk 1]
```diff
@@ -102,25 +87,12 @@ def plot_dates_values(data):
...
-
-# Normalize `value` and save the mean and std we get,
+# Normalize and save the mean and std we get,
 # for normalizing test data.
-training_value, training_mean, training_std = normalize(training_value)
-len(training_value)
+training_mean = df_small_noise.mean()
+training_std = df_small_noise.std()
+df_training_value = (df_small_noise - training_mean) / training_std
+len(df_training_value)
```
Comment (fchollet) [inline examples/timeseries/timeseries_anomaly_detection.py:95]: While we're at it -- please print a complete sentence here, e.g. "Number of training samples: %d"

File: examples/timeseries/timeseries_anomaly_detection.py [hunk 2]
```diff
@@ -137,10 +109,10 @@ def create_sequences(values, time_steps=TIME_STEPS):
         output.append(values[i : (i + time_steps)])
     # Convert 2D sequences into 3D as we will be feeding this into
     # a convolutional layer.
-    return np.expand_dims(output, axis=2)
+    return np.stack(output)
```
Comment (fchollet) [inline examples/timeseries/timeseries_anomaly_detection.py:112]: Why this change?

Comment (RobAltena) [reply examples/timeseries/timeseries_anomaly_detection.py:112]: It took me a few minutes to figure out what was happening here. Eventually the light went on: "It is stacking the training data". When making the changes this came naturally. The result is the same so fair enough to question the change. I have moved and updated the comments.
